package.json

```diff
@@ -10,8 +10,10 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "autoprefixer": "^9",
+    "axios": "^0.21.1",
```

Comment:
do we really need axios?

Reply:
removed on f7c6c4db3d861f8fe7b7e03254be4dddb6d24911